assets/i18n/es.po

```diff
@@ -0,0 +1,720 @@
+#: assets/page/Help.svelte:38
+msgid "\"#\" will show the matching conversation names."
+msgstr "\"#\" mostrará los nombres de las conversaciones que coincidan."
```

Comment:
Although the English original already has "names" in plural, I get the feeling "el nombre de las conversaciones" would sound more natural in Spanish.

Reply:
Thanks! Sounds better indeed. Already resolved!